test/moduleId.html

```diff
@@ -5,6 +5,9 @@
 		<title>QUnit ModuleId Test Suite</title>
 		<link rel="stylesheet" href="../dist/qunit.css">
 		<script src="../dist/qunit.js"></script>
+		<script>
+			QUnit.warning.off();
+		</script>
```

Comment:
I need to investigate why they're being triggered here. I'll give some feedback after this.